src/dyn/dyn_variables.cpp

```diff
@@ -615,7 +643,7 @@ vector< vector<double> > dyn_variables::get_fssh3_errors(){
 vector<double> dyn_variables::get_fssh3_average_errors(){
   int itraj, k;
   vector<double> res(5,0.0);
-  for(itraj=0;itraj<ntraj;itraj++){
+  for(itraj;itraj<ntraj;itraj++){

```

Comment:
it is safer to have explicit initialization `itraj=0`

Reply:
wait, also - why do you change this function at all?

Reply:
I think that was a mistake, will be more careful